starter-code/index.js

```diff
@@ -1,17 +1,46 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    this.length++;
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos >= this.items.length) {
+      throw new Error("OutOfBounds");
+    }
+    return this.items[pos];
+  }
 
-  max() {}
+  max() {
```

Comment:
I think for max and min you should return the highest/lowest number in the array. right now this works because the array is sorted but maybe safer to use Math.max() Math.min() :)